lib/Froxlor/Database/DbManager.php

```diff
@@ -87,6 +87,8 @@ public function createDatabase($loginname = null, $password = null, $last_accnum
 			while (in_array($username, $allsqlusers)) {
 				$username = $loginname . '-' . substr(md5(uniqid(microtime(), 1)), 20, 3);
 			}
+		} elseif (strtoupper(Settings::Get('customer.mysqlprefix')) == 'DBNAME') {
```

Comment:
This would lead to the same username being used for multiple databases which might be an unwanted behaviour. An idea would be to use the random-username-generation-code from the above case (mysqlprefix == 'RANDOM') so each database has its unique username

Reply:
This is not correct because the user that is being created corresponds to the name of the database. Each database has its own user who has the same name as the database itself.